<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShelfPage</class>
 <widget class="QWidget" name="ShelfPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>717</width>
    <height>637</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="hLayout_2">
   <item>
    <layout class="QVBoxLayout" name="vLayout_1">
     <item>
      <layout class="QHBoxLayout" name="hLayout_1">
       <item>
        <widget class="QPushButton" name="back_btn">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="add_new_album_btn">
         <property name="text">
          <string>Add new album</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="connect_album_btn">
         <property name="text">
          <string>Connect  album</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="disconnect_album_btn">
         <property name="text">
          <string>Disconnect  album</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListWidget" name="albums_list"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="albums_info_stack">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>3</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QWidget" name="empty_page"/>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
